action_list_sample/action_list.js

```diff
@@ -3,17 +3,37 @@ import {
   StyleSheet,
   Text,
   ListView,
-  View
+  View,
+    ScrollView
 } from 'react-native';
 
+import { Button } from 'react-native-elements';
+import Panel from './panel';
+
 var lib = require("./commons.js")
 
 export default class ActionList extends Component {
   renderList(rowData) {
     return (
-      <View>
-      <Text> {rowData.id} <Text>{rowData.title}</Text> <Text>{rowData.status}</Text> </Text>
-      </View>
+        <View>
+        <ScrollView style={styles.container}>
+          <Panel title={rowData.title}>
+            <Text>{rowData.processType}</Text>
+            <Text>{rowData.initiator}</Text>
+            <Text>{rowData.lastUpdated}</Text>
+            <Text>{rowData.status}</Text>
+            <Text>{rowData.actions}</Text>
+        <View style={{width: 300, flex: 1,
+         flexDirection:'row',}}>    
+        <Button
+                title='Take Action'  /> 
+          
+            <Button
+                title='Route Log' style={styles.routeButton}/> 
```

Comment:
1. `style` should appear on the next line if you're breaking up the properties (I'm not sure it's necessary) https://github.com/airbnb/javascript/tree/master/react#alignment
1. There should be a space before the `/` https://github.com/airbnb/javascript/tree/master/react#alignment